sufia-models/app/models/sufia/works/curation_concern/with_generic_files.rb

```diff
@@ -0,0 +1,17 @@
+module Sufia::Works
+  module CurationConcern
+    module WithGenericFiles
+      extend ActiveSupport::Concern
+
+      included do
+        # This used to have a hasFile relation when in hydra-works.  That does not seem to exist so I am using hasPart instead
```

Comment:
Is this comment worthy of a TODO or a new ticket, or do we already have a ticket that wraps this up?